docs/reference/robot.md

```diff
@@ -2358,4 +2358,15 @@ The message is sent using the `webots.window("<robot window name>").send` method
 The `wb_robot_window_send` and `wb_robot_wwi_send_text` functions allow a robot controller to send a message to a JavaScript function running in the HTML robot window.
 The message is received using the `webots.window("<robot window name>").receive` method of the Webots JavaScript API.
 
+The `wb_robot_wwi_send_text` returns the first message present in the buffer of received messages and moves its reading head to the next one. To read the full buffer, you can proceed like:
+```
+const char * message = wb_robot_wwi_receive_text();
+while (message){
+  // to something with the message.
+
+  message = wb_robot_wwi_receive_text();
+}
+```
```

Comment:
A simpler version could be:
```C
const char *message;
while (message = wb_robot_wwi_receive_text()) {
  // do something with each message
}
```
I believe this simpler version should be used in the code also.